app/(auth)/login/page.tsx

```diff
@@ -2,50 +2,32 @@
 
 import { useState } from 'react';
 import { useRouter } from 'next/navigation';
-import supabase from '../../../api/supabase/createClient';
+import { AuthProvider, useAuth } from '../../utils/AuthProvider';
 
-export default function Login() {
+export default function LoginLayout() {
+  return (
+    <AuthProvider>
+      <Login />
+    </AuthProvider>
+  );
+}
```

Comment:
@rachaelch3n follow up on these comments about moving  AuthProvider to `app/layout.tsx`, and out of this page's layout.